tests/scripts/localPathPV.sh

```diff
@@ -10,7 +10,10 @@ if [ ! -b "${test_scratch_device}" ] ; then
   exit 1
 fi
 
-lsblk
+# wipe the full disk just in case it was not cleaned up correctly or fully
+sudo dd if=/dev/zero of="$test_scratch_device" bs=1M oflag=direct
```

Comment:
We're still getting the issue creating OSDs, so perhaps it isn't related to CI cleanup